docs/installation.rst

```diff
@@ -5,14 +5,13 @@
 Installation
 ============
 
-To install |project_name|, you need `Python 3`_ and `pip`_ installed. You
-can then either
-`download it directly
+To install Borg-Import, you need `Python 3`_ and `pip`_ installed. You can then either `download borg-import directly
 <https://github.com/borgbackup/borg-import/archive/master.zip>`_
-or clone it from github_.
-Extract the content, open a terminal in its root folder and
-execute the following to install it via pip:
+or clone it from the `repo`_.
+Extract the downloaded .zip if necessary.
+Open a terminal in the borg-import directory and execute the following to install the program via pip:
 :code:`pip install --user -e .`
 
-If you have */home/user/.local/bin/* in your ``PATH`` variable,
-you can then start using |project_name|.
+If you have */home/user/.local/bin/* in your ``PATH`` variable, you can then start using Borg-Import.
+Otherwise, you will need to add *.local/bin/* to your ``PATH`` via adding ``export PATH=$PATH:/home/user/.local/bin/``
+to your .profile file if you are using a POSIX-based shell. Don't forget to :code:`source ~/.profile` when you're done.
```

Comment:
https://linuxtect.com/zsh-profile-file-profile-zprofile-zshrc/ seems like the instructions would not work for macOS.

Can we just keep 17/18 as is and assume all users can find out elsewhere how to set PATH for their OS?